src/windows-manager.cpp

```diff
@@ -746,6 +746,27 @@ namespace graphics {
         }
     }
 
+  bool WindowsManager::setTexture (const char* nodeName,
+				   const char* filename)
+  {
+    NodePtr_t node (getNode (nodeName, true));
+    LeafNodeFacePtr_t faceNode (boost::dynamic_pointer_cast <LeafNodeFace>
```

Comment:
For a record of our discussion: method `setTexture` of `LeafNodeFace` should be moved to `Node` as many children of `Node` implement it.